### ISSUE TITLE
CycleGAN is bypassing information to minimize the reconstruction error

### ISSUE THREAD

**Issue by jgciudad** [junyanz/pytorch-CycleGAN-and-pix2pix#1429]
Hi, the results I obtained in the facade dataset look quite strange.

To me, it looks like if the CycleGAN is keeping the information in the synthetic generated images, so then it can get a perfect reconstruction of the input images, and is completely forgetting about optimizing the discriminators error. Note that even though the reconstruction of the facade looks identical to the original image, the following picture is with data from the test set, so is not an overfitting issue.

Does anybody has a hint of what can be happening? I saw [this ](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/issues/1184#issuecomment-989188534 )answer from #1184, which I don't know if could be somehow related. 

Thanks!

![imagen](https://user-images.githubusercontent.com/59373672/171160980-06d08fe2-2bb4-4e68-b8bb-937741a443bf.png)

**Comment by junyanz** [comment]
This limitation is documented by a well-written article "[CycleGAN, a Master of Steganography](https://arxiv.org/abs/1712.02950)". I am wondering if you are using the default parameters for the facades results. We have obtained better results compared to yours.
